docs-2.0/3.ngql-guide/10.tag-statements/2.drop-tag.md

```diff
@@ -0,0 +1,29 @@
+# DROP TAG
+
+```ngql
+DROP TAG [IF EXISTS] <tag_name>
+```
+
+`DROP TAG` drops a tag with the given name in a graph space. You must have the `DROP` privilege for the graph space. (TODO: privilege doc) To drop a tag in a specific graph space, you must use the graph space first.
+
+Before you drop a tag, make sure that the tag does not contain any indexes or TTLs. If a tag contains any indexes or TTLs, a conflict error occurs when you drop it.
+
+A vertex can have one or more tags. If a vertex has only one tag, after you drop it, the vertex **CANNOT** be accessible. But its edges are available. The vertex is deleted in next compaction. If a vertex has multiple tags, after you drop one of them, the vertex is still accessible. But all the properties defined by this dropped tag are not accessible.
```

Comment:
1. The vertex is deleted in next compaction. => The vertex is deleted in **the** next compaction.
2. I would recommend using an unordered list to show the two conditions: a vertex has only one tag and a vertex has multiple tags. Much clearer.
3. Please double confirm if we still allow hanging edges. I remember the plan was to remove edges together with the vertices if there is only one tag on the vertices.

Reply:
confirmed already